sdk/communication/Azure.Communication.Sms/src/Models/SmsModelFactory.cs

```diff
@@ -8,11 +8,14 @@ namespace Azure.Communication.Sms.Models
     /// </summary>
     public static class SmsModelFactory
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="SendSmsResponse"/> class.
-        /// </summary>
-        /// <param name="messageId"> The identifier of the outgoing SMS message. </param>
-        public static SendSmsResponse SendSmsResponse(string messageId)
-            => new SendSmsResponse(messageId);
+        /// <summary> Initializes a new instance of SmsSendResult. </summary>
+        /// <param name="to"> The recipients&apos;s phone number in E.164 format. </param>
+        /// <param name="messageId"> The identifier of the outgoing SMS message. Only present if message processed. </param>
+        /// <param name="httpStatusCode"> HTTP Status code. </param>
+        /// <param name="repeatabilityResult">The result of a repeatable request with one of the case-insensitive values accepted or rejected. </param>
+        /// <param name="succeeded"> Flag to check if message processing succeeded or not. </param>
+        /// <param name="errorMessage"> Optional error message in case of 4xx or 5xx errors. </param>
+        public static SmsSendResult SmsSendResult(string to, string messageId, int httpStatusCode, SmsSendResponseItemRepeatabilityResult? repeatabilityResult, bool succeeded, string errorMessage)
```

Comment:
we should probably change `repeatabilityResult` to string when `SmsSendResponseItemRepeatabilityResult` becomes internal. Or does it make sense to not receive it here and use a default? We do not expose the repeatability concept anywhere. Let's ask if this should be string or non-existent in the ApiView